manager/resolver/maven/src/main/java/org/jboss/forge/furnace/manager/maven/util/MavenRepositories.java

```diff
@@ -26,19 +26,17 @@
  */
 public class MavenRepositories
 {
-   private static final String MAVEN_CENTRAL_REPO = "http://repo1.maven.org/maven2";
+   public static final String MAVEN_CENTRAL_REPO = "http://repo1.maven.org/maven2";
```

Comment:
sure, I made it public for possible future use, but it could be done later if is needed.